app/javascript/components/App.js

```diff
@@ -25,12 +25,71 @@ class App extends Component{
     super(props)
     this.state = {
       cards: mockCards,
-      bindersleeves: mockBinders,
+      bindersleeves: [],
       users: mockUsers,
-      rankings: mockRankings
+      rankings: mockRankings,
+      isUserEligible: false
     }
   }
 
+  componentDidMount(){
+    if (this.props.current_user) {
+      this.dailyCardEligibilityCheck(this.props.current_user.id)
```

Comment:
Is this happening as expected? The asynchronous behavior here leads me to think there might be a gap in how eligibility is being used